client/src/components/AuthMenu/AuthMenu.tsx

```diff
@@ -40,6 +45,9 @@ const AuthMenu = (): JSX.Element => {
         }}
         getContentAnchorEl={null}
       >
+        <MenuItem>
+          <Link to={to}>Profile</Link>
```

Comment:
Can you change this link to not be underlined and blue?

Reply:
Yes, I will get that done right away.